Documentation/ReleaseNotes.md

```diff
@@ -38,13 +38,67 @@ If importing the [Mixed Reality Toolkit NuGet packages](MRTKNuGetPackage.md), th
 
 ### Upgrading projects to 2.4.0
 
-#### Custom controller classes
+The 2.4.0 release has some changes that may impact application projects. Breaking change details, including mitigation guidance, can be found in the [**Updating 2.3.0 to 2.4.0**](Updating.md#updating-230-to-240) article.
 
-Custom controller classes previously had to define `SetupDefaultInteractions(Handedness)`. This method has been made obsolete in 2.4, as the handedness parameter was redundant with the controller class' own handedness. The new method has no parameters. Additionally, many controller classes defined this the same way (`AssignControllerMappings(DefaultInteractions);`), so the full call has been refactored down into `BaseController` and made an optional override instead of required.
+> [!NOTE]
+> At this time, it is not supported to switch between using .unitypackage files and NuGet.
+
+**Updating using .unitypackage files**
+
+For the smoothest upgrade path, please use the following steps.
```

Comment:
On a similar note, how would we feel about making the ReleaseNotes.md (and associated files) valid only for the release that's being published? We publish the docs for each release separately, so, if I'm looking at the 2.3 docs, I might not care about how to update from 2.0 to 2.1. I can go to the 2.1 docs for that.

It feels like this doc will eventually become so long that it feels intimidating, especially if I only care about the release I'm looking at

Reply:
that makes sense, though might be something we always have to be aware of when merging between release / dev.  we could also change the release notes menu item into a folder ref and have each release in a separate file - that way we wouldn't have a problem merging and still have all the data available without overwhelming the users

Reply:
Let's split this suggestion out as well.